Applications/ApplicationsLib/UncoupledProcessesTimeLoop.h

```diff
@@ -29,7 +30,12 @@ template<typename Matrix, typename Vector>
 class UncoupledProcessesTimeLoop
 {
 public:
-    bool loop(ProjectData& project, std::string const& outdir);
+    bool loop(ProjectData& project);
+
+    explicit UncoupledProcessesTimeLoop(
```

Comment:
Constructors and Destructor should come before declaration of methods

see http://ufz.github.io/styleguide/cppguide.xml?showone=Declaration_Order#Declaration_Order